docs/build.md

```diff
@@ -2,40 +2,84 @@
 
 Dependency | Version | Required           | Runtime
 ---        | ---     | ---                | ---
-Kokkos     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
+[Kokkos](https://github.com/kokkos/kokkos)     | 3.1.00  | :heavy_check_mark: | :heavy_check_mark:
 MPI        | 2       |                    | :heavy_check_mark:
 CMake      | 3.12    | :heavy_check_mark: |
 
-# Installation
+# Build and installation
 
-## Installation using Spack
+## CMake
 
-ArborX can be installed using [Spack](https://github.com/spack/spack) package manager:
+Assuming that Kokkos is installed in `$KOKKOS_INSTALL_DIR`, configure ArborX
+```CMake
+ARGS=(
+    -D CMAKE_BUILD_TYPE=RELWITHDEBINFO
+    -D CMAKE_INSTALL_PREFIX="$ARBORX_INSTALL_DIR"
+    -D ARBORX_ENABLE_MPI=ON
+    -D CMAKE_PREFIX_PATH="$KOKKOS_INSTALL_DIR"
+    -D CMAKE_CXX_COMPILER="$KOKKOS_INSTALL_DIR/bin/nvcc_wrapper"
+    -D CMAKE_CXX_EXTENSIONS=OFF # required by Kokkos
```

Comment:
That's fine but I believe that's a CUDA only thing.

Reply:
I trust you on this. I have no strong opinion whether we should mention it. It the configuration fails, I believe Kokkos will spell it out.